QuizApp/App/Services/SecureStorage.swift

```diff
@@ -5,10 +5,30 @@ protocol SecureStorageProtocol {
 
     func save(_ token: String)
 
+    func getToken() throws -> String
```

Comment:
functions that return a value and have no arguments should be computed properties